### ISSUE TITLE
Feature request: add "open in" in iOS action menu

### ISSUE THREAD

**Issue by Doc-Steve** [strongbox-password-safe/Strongbox#244]
### Situations:
1.  Sometimes you want to create an Strongbox entry while browsing a website even without the password entry yet or 

2. the password entry is some kind of special field with on screen keyboard without keyboard entry (some banks use that).
In these situations you have to leave the browser and open Strongbox and search manually for the site.

### Proposed solution:
Add Strongbox to the "action menu" (the menu where you e.g. can print), like e.g. 1Password does.
Opening an URL with the  action menu will open the search menu in Strongbox like when opening Strongbox from the keybord.

**Comment by mmcguill** [comment]
If you have a sample video for this it would be greatly appreciated as I don't know exactly what this should look like

**Comment by Doc-Steve** [comment]
> If you have a sample video for this it would be greatly appreciated as I don't know exactly what this should look like

See this YouTube video: https://m.youtube.com/watch?v=Jev4E22l04s&feature=youtu.be

Hope, this helps.

**Comment by georgesnow** [comment]
This is similar to the request I had made. I will post the number in here later. I don’t remember the ticket number

**Comment by mmcguill** [comment]
Cool, got it.

**Comment by Doc-Steve** [comment]
> This is similar to the request I had made. I will post the number in here later. I don’t remember the ticket number

Yes, you are right. I think I found it: it's "Share Extension for Form Filling #85", right?

I would also like the "form autofill" of your request. My experience with the "form autofill" of 1Password is unfortunately not great. Maybe that's because I often fill out forms on non-English pages and then the field matching is not good. So I end up in copying field by field. That's why I openend the other feature request for drag&drop.

Opening an item for viewing when browsing on a webpage would also help me often a lot. Sometimes I only need one ore two fields of an entry: then this solution above would be great for that.

I think it's not a either/or decision: in a good software there should sometimes be multiple ways to accomplish something, depending on your workflow and preferences. At the end I would love to see both requests my "action view" and your "action fill form" come to life :)

I added a link to your request in the description above.

**Comment by georgesnow** [comment]
I agree the form filling even is English was hit or miss. Credit cards worked pretty well. Which is the thing I miss most recently from 1Password. 

But the ability to get to the action menu and easily copy items from entry would be great. That would allow me copy the credit card number without having jump back and forth from the strongbox app and safari or even an app
